pylintrc

```diff
@@ -70,7 +70,8 @@ disable=
     standarderror-builtin,import-star-module-level,old-octal-literal,
     metaclass-assignment,reduce-builtin,indexing-exception,
     parameter-unpacking,long-suffix,cmp-method,using-cmp-argument,
-    useless-suppression
+    useless-suppression,import-error,
+    empty-docstring,missing-docstring,redefined-outer-name
```

Comment:
I would keep attribute-defined-outside-init. It saved me from lots of bugs with undefined attributes in widgets, and only seldom did I have to add something to `__init__` that really didn't belong there.